README.md

```diff
@@ -56,6 +56,14 @@ You can run the unit tests by executing:
 npm run test
 ```
 
+To run the e2e tests, run:
+
+```bash
+npm run test:e2e
+```
```

Comment:
Should we add a note that you might have to run `npx playwright install` to get the browsers installed if `test:e2e` fails?